packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -274,10 +274,11 @@ export class MLSService {
       }),
     );
 
-    let response;
-    if (coreCryptoKeyPackagesPayload.length !== 0) {
-      response = await this.addUsersToExistingConversation(groupIdBytes, coreCryptoKeyPackagesPayload);
-    }
+    const response =
+      keyPackages.length > 0
+        ? await this.addUsersToExistingConversation(groupIdBytes, keyPackages)
+        : // If there are no clients to add, just update the keying material
```

Comment:
```suggestion
        : // If there are no clients to add, just update the keyring material
```

Reply:
Not a typo ;)

Reply:
Yup, became clear after a little googling and looking at the rest of the comments. Learning bit by bit :)